doc/faq.rst

```diff
@@ -100,12 +100,31 @@ See :ref:`adding_graphical_models`.
 Will you add GPU support?
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-No, or at least not in the near future. The main reason is that GPU support
-will introduce many software dependencies and introduce platform specific
-issues. scikit-learn is designed to be easy to install on a wide variety of
-platforms. Outside of neural networks, GPUs don't play a large role in machine
-learning today, and much larger gains in speed can often be achieved by a
-careful choice of algorithms.
+A limited but growing :ref:`list of scikit-learn estimators
+<array_api_supported>` can already run on GPUs if the input data is provided
+as a PyTorch or CuPy array and if scikit-learn has been configured to accept
+such inputs as explained in :ref:`array_api`.
+
+Most estimators that rely on NumPy for their computationally intensive operations
+can be considered for Array API support and therefore GPU support.
+
+However, not all scikit-learn estimators are amenable to running on GPUs via
```

Comment:
I'm wondering if we need the level of detail of this paragraph. Maybe we can just state that "However, not all scikit-learn estimators can be efficiently implemented using the Array API. Adding efficient GPU support to these estimators would require designing and adopting a more flexible extension system for scikit-learn. This possibility is currently being considered."

I'm thinking that we might not need to explain why this is so in a FAQ entry. I assume most people looking at the FAQ are here for a quick and simple answer. The longer we make the answer the more things there are that people could misunderstand or maybe even disagree with (and then start arguing over whether or not they are true).

What do you think?

Reply:
As you prefer. The me from 3 years ago (before I had the opportunity to learn more about those matters) would have appreciated a finer level of details to understand why the short version you propose is not a completely arbitrary statement.

Reply:
Fair point. Not sure I have a strong opinion, I like shorter text but I can see that it is interesting to know why the short version is what it is.

Reply:
@adrinjalali do you want to be the tie breaker on this?

Reply:
It was me, I would go for even a longer version. So I'd say this is a good balance.